addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -40,10 +40,33 @@ def _domain_task_id(self):
         return []
 
     task_id = fields.Many2one(
+<<<<<<< HEAD
+        'project.task',
+        'Task',
+        compute='_compute_task_id',
+        store=True,
+        readonly=False,
+        index=True,
+        domain="[('company_id', '=', company_id),"
+                "('project_id.allow_timesheets',"" '=', True),"
+                "('project_id', '=?', project_id)]")
+=======
```

Comment:
coucou conflict marker :laughing:

Reply:
Yep, but this one is already fixed I think ;-)

Reply:
But in two differents commits. Commit https://github.com/odoo/odoo/pull/45818/commits/f8d291dcca93820c8fb6ac7cc7c4b15be5a71ce4 introduces it and 2f1d69d removes it. We avoid introducing such noise in the commit history. Especially for conflict marker that should never appear in the history ;)